packages/dcos-image/extra/dcos-shell

```diff
@@ -2,6 +2,7 @@
 set -o errexit -o nounset -o pipefail
 
 source /opt/mesosphere/environment.export
+source /opt/mesosphere/etc/proxy.env
```

Comment:
IIRC this script doesn't have export statements, does this play well with subprocesses in the shell session? I think they might not inherit from the proxy env vars.

@branden and I were thinking of creating a `proxy.env.export` file with the export statements, but if just sourcing the file is enough then that'd be better.

Reply:
Yes! Thank you @bamarni. I did not notice that the `proxy.env` did not have its own `export` statements. In this case, this solution partially complete. I am providing a change that will address this.